.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,20 @@
+name: Continuous Integration
+on: [pull_request]
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Checkout code"
+      uses: actions/checkout@v2
+    - name: "Create env file"
+      run: |
+        touch .env
```

Comment:
I could be wrong, but iirc you don't actually need to `touch .env` first before redirecting input to the file :)

Reply:
Also, I do enjoy the slightly more novel approach you have here of echoing your secrets into the env file :)

However, you should be able to just set the secrets in the environment with
```
env:
  super_secret: ${{ secrets.SuperSecret }}
  another_secret: ${{ secrets.AnotherSecret }}
```

The env file is usually there just as an alternative for dev environments to setting your environment variables :)